src/langs/en/messages.ts

```diff
@@ -28,6 +28,18 @@ export default {
     'One item has too many memberships for the operation to proceed',
   [FAILURE_MESSAGES.MEMBER_CANNOT_ACCESS]:
     'A user does not have access to an item',
+  [FAILURE_MESSAGES.MEMBER_ALREADY_SIGNED_UP]:
+    'You already has an account, we will send you an email to login',
+  [FAILURE_MESSAGES.MEMBER_NOT_SIGNED_UP]: 'No account found for this email',
+  [FAILURE_MESSAGES.MEMBER_WITHOUT_PASSWORD]:
+    'This account has no password configured',
+  [FAILURE_MESSAGES.INCORRECT_PASSWORD]:
+    'The submitted password is incorrect for this account',
+  [FAILURE_MESSAGES.TOKEN_EXPIRED]: 'Your session token has expired',
```

Comment:
Not sure if we should mention "token" as it is an implementation detail. Would it be simpler for users to have "Your session has expired" and "Your session is invalid"?